src/index.jsx

```diff
@@ -1,14 +1,42 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import { render } from 'react-dom';
 import './style.css';
 
+import AnimalList from "./components/AnimalList";
+import AnimalDetail from "./components/AnimalDetail";
+
+
 const App = () => {
+
+  const [animals, setAnimals] = useState([]);
+  const [chosenAnimal, setChosenAnimal] = useState({});
```

Comment:
tady bych pro žádné zvíře asi preferoval null/undefined než prázdný objekt. De facto říkáš že "není vybrané zvíře" a ne "je vybrané zvíře bez názvu". Ale to je taky jen sémantická drobnost